pkg/process/checks/chunking.go

```diff
@@ -0,0 +1,168 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package checks
+
+import (
+	"unsafe"
+
+	model "github.com/DataDog/agent-payload/v5/process"
+)
+
+type payloadList interface {
```

Comment:
nit: what do you think about giving a brief description of each interface, its methods and how it's intended to be used?

Reply:
Yes, good idea, I think it's not obvious.